### PR TITLE
Update Class Attention Image Transformers example for Keras 3.

### PR DESCRIPTION
**Pull request by hertschuh**

### CODE REVIEW

File: examples/vision/md/cait.md [hunk 1]
```diff
@@ -143,9 +150,9 @@ class StochasticDepth(layers.Layer):
     def call(self, x, training=False):
         if training:
             keep_prob = 1 - self.drop_prob
-            shape = (tf.shape(x)[0],) + (1,) * (len(tf.shape(x)) - 1)
-            random_tensor = keep_prob + tf.random.uniform(shape, 0, 1)
-            random_tensor = tf.floor(random_tensor)
+            shape = (ops.shape(x)[0],) + (1,) * (len(x.shape) - 1)
+            random_tensor = keep_prob + ops.random.uniform(shape, 0, 1)
```
Comment (fchollet) [inline examples/vision/md/cait.md:154]: Please add `seed=self.seed_generator` and create `self.seed_generator` in the constructor. It's a best practice

**Review by fchollet — COMMENTED** [review]
Looking good, thank you!

File: examples/vision/md/cait.md (hunk 1 above)
Comment (hertschuh) [reply examples/vision/md/cait.md]: Added, thanks for the review!

**Review by fchollet — APPROVED** [review]
LGTM, thanks!

**Pull request merged**